exercises/concept/salary-calculator/.meta/design.md

```diff
@@ -21,6 +21,15 @@ This exercise's prerequisites Concepts are:
 
 ## Analyzer
 
-This exercise could have the following rule added to the analyzer:
+This exercise could benefit from the following rules in the [analyzer]:
 
-- Verify that ternary operators were used.
+- `essential`: If the student did not use `ternary operators` in the `salaryMultiplier` method, instruct them to do so.
+- `essential`: If the student did not use `ternary operators` in the `bonusMultiplier` method, instruct them to do so.
+- `essential`: If the student did not use `ternary operators` in the `finalSalary` method, instruct them to do so.
```

Comment:
Let's collapse this into one comment, that may also make it more clear that the student should not receive this comment 3 times when not using ternaries at all.

Reply:
Sure, you're right!

Reply:
Done!